console.js

```diff
@@ -0,0 +1,37 @@
+const repl = require('repl'),
+  fs = require('fs'),
+  pjson = require('./package.json'),
+  orm = require('./app/orm'),
+  testFolder = './tests/';
+
+const convertFunctionToAsync = f => {
+  return async (...args) => {
+    const result = await f(...args);
+    console.log(result);
+  };
+};
+
+const convertObjectFunctionsToAsync = serviceMethods => {
+  const asyncServiceMethods = {};
+  Object.keys(serviceMethods).forEach(key => {
+    asyncServiceMethods[key] = convertFunctionToAsync(serviceMethods[key]);
```

Comment:
What if `serviceMethods[key]` is not a function? Like a constant we are exporting. Won't it fail?
I'd check that and if its type is not a function, then we just assign it.